interface/i18n/mist.de.i18n.json

```diff
@@ -49,9 +49,9 @@
                 "startMining": "⛏ Mining starten (nur auf Testnetz)",
                 "stopMining": "⛏ Mining stoppen",
                 "externalNode": "externer Softwareknoten aktiv",
-                "nodeMode": "Chain download",
-                "fullNode": "Store full blockchain",
-                "lightNode": "Use light Node (experimental!)"
+                "nodeMode": "Chain herunterladen",
+                "fullNode": "Komplette Blockchain speichern",
+                "lightNode": "Vereinfachte Blockchain speichern (experimentell!)"
```

Comment:
"Light Node", should stay as its not a simple blockchain, its a "light node" (no translation for now)